packages/express-did-auth/src/jwt-utils.ts

```diff
@@ -0,0 +1,59 @@
+import { createJWT, JWTVerified, verifyJWT } from 'did-jwt'
```

Comment:
Is this a `utils`? It is more part of the core than any other module... In fact, most of the critical logic is happening here.

Reply:
Maybe the name is not too declarative 😏 
I wanted to extract the logic to another file because it used in several factories